protos/escrow/escrow.proto

```diff
@@ -88,6 +90,7 @@ message SubmitContractResult {
   repeated Escrow escrow = 2;
   bytes escrow_address = 3;
   google.protobuf.Timestamp escrow_signed_time = 4 [(gogoproto.stdtime) = true];
+  ledger.SignedChannelState buyer_channel_state = 5; // escrow signed then send to buyer to sign
```

Comment:
Suggest reserve repeated host_collateral_channel here

Reply:
I talked to Eric yesterday, now we have two options for how to use the collateral:
1. seller sign and pay collateral for each contract
2. seller deposit once before start serving, escrow record in database, when submit contract escrow only verify if the seller has enough collateral in escrow database.
Eric said we don't need to consider collateral for now so I didn't put host_collateral_channel here.